crates/sui-framework/sources/math.move

```diff
@@ -133,6 +133,23 @@ module sui::math {
         (res as u128)
     }
 
+    // Calculate the square root of an usigned integer using Babylonian method.
```

Comment:
It's not clear if this method behaves exactly the same for u128 and u64 as with other sqrt functions in this file. We'll need fuzzy tests to support it and we should ensure they are always aligned.